code/game/gamemodes/events/hidden_events/kaiser.dm

```diff
@@ -49,6 +49,9 @@ It will also bring a hoard of roaches with it.
 		failure = TRUE
 
 /datum/event/kaiser/start()
+	if(prob(1))			// occulus kaiserin start edit
+		spawn_kaiserin()
```

Comment:
Why are we adding a whole new proc here?

Reply:
Cause I'm a bad coder, and I glanced at the spawn_mobs proc at 2 am and went 'durrr, I can't adjust that to work.'
Fixed.